frontend/src/app/testResults/uploads/UploadForm.tsx

```diff
@@ -394,7 +382,7 @@ const Uploads = () => {
               <li className="usa-process-list__item margin-bottom-1em">
                 <p className="usa-process-list__heading">
                   Visit the{" "}
-                  <LinkWithQuery to="/results/upload/submit/guide">
+                  <LinkWithQuery to={uploadGuideLocation ?? ""}>
```

Comment:
should we just display text if there is no `uploadGuideLocation`?

Reply:
I will make uploadGuideLocation a required property, but since in reality they should always have values I don't think we need to particularly handle that case. If you think otherwise let me know!

Reply:
Sounds good! thank you!!